openstudiocore/src/utilities/core/UUID.i

```diff
@@ -12,16 +12,13 @@
 namespace openstudio{
 
   #ifdef SWIGRUBY
-    %rename("nil?") isNull();
+    %rename("nil?") is_nil();
```

Comment:
Plus, I of course verified that all existing ruby uuid related unit tests still pass.